service/ExchangeService.js

```diff
@@ -7,19 +7,31 @@ var btcPrice   = null;
 var krwUsd     = null;
 var log = new Logger(AppConfig.logLevel);
 
-
+var handleError = function(message, exchange, errortype){
+	var errorObject = new Error();
+		errorObject.message = message;
+		errorObject.exchange = exchange;
+		errorObject.errorType = errortype;
+	return errorObject;
+}
 var fetchExchangeData = function(exchange, url, callback){
 	request.get(url, function (err, response, body) {
 		if ( !err && response.statusCode == 200 ){
 			try {
 				callback(null, JSON.parse(body));
-			}catch (e) {
-				return callback('ERROR parsing response from ' + exchange + '. Details: ' + e + '\nResponse::' + body);
+			}catch (e){
+				var parseError = 'ERROR parsing response from ';
+				log.debug(parseError + exchange + '. Details: ' + e + '\nResponse::' + body);
+				return callback(handleError(parseError, exchange, e), null);
 			}
 		}else if ( err ){
-			return callback('ERROR: fetching data from ' + exchange + ' (' + url + ') Details: ' + err + '. BODY=' + body, null);
+			var errorFetching = 'ERROR: fetching data from ';
```

Comment:
I think "error" prefix in error messages is excessive

Reply:
@shumkov we using this variable in logger as well it could be great to highlight these kinds of messages with Error prefix in our logger

Reply:
I see, in this case we can add prefix only for logger:
```js
var errorMessage = "Can't fetch data";
log.debug('ERROR: ' + errorMessage + '...');
return callback(handleError(errorMessage, exchange, err), null);
```